app/assets/javascripts/components/story/StoryItem.js

```diff
@@ -1,206 +1,14 @@
 import React from 'react';
 import classname from 'classnames';
 import PropTypes from 'prop-types';
-import StoryPopover from './StoryPopover';
-import StoryDescriptionIcon from './StoryDescriptionIcon'
-import {
-  classIconRule,
-  iconRule,
-  isStoryNotEstimated,
-  IsShow,
-  estimateRule,
-  labelSplit,
-  isRelease,
-} from '../../rules/story';
+import CollapsedStory from './CollapsedStory/CollapsedStory';
```

Comment:
Suggestion: what do you think about having the component code inside of an `index.js` file so whenever you import it you end up with `import Component from './Component';` which avoids that duplicate name?